praw/models/reddit/subreddit.py

```diff
@@ -6,10 +6,11 @@
 from urllib.parse import urljoin
 
 from prawcore import Redirect
+import socket as socket_module
```

Comment:
This is a system package right? If so, can we move it to the top level import section?